implementations/rust/ockam/ockam_command/src/credential/show.rs

```diff
@@ -0,0 +1,54 @@
+use clap::{arg, Args};
+use colorful::Colorful;
+use ockam::Context;
+use ockam_identity::{credential::Credential, IdentityIdentifier};
+
+use crate::{
+    credential::validate_encoded_cred, util::node_rpc, vault::default_vault_name, CommandGlobalOpts,
+};
+use anyhow::anyhow;
+
+#[derive(Clone, Debug, Args)]
+pub struct ShowCommand {
+    #[arg()]
+    pub credential_name: String,
+
+    #[arg(default_value_t = default_vault_name())]
+    pub vault: String,
+}
+
+impl ShowCommand {
+    pub fn run(self, opts: CommandGlobalOpts) {
+        node_rpc(run_impl, (opts, self));
+    }
+}
+
+async fn run_impl(
+    ctx: Context,
+    (opts, cmd): (CommandGlobalOpts, ShowCommand),
+) -> crate::Result<()> {
+    let cred_state = opts.state.credentials.get(&cmd.credential_name)?;
```

Comment:
Will address in other PR, because that's where I made changes to retrieving the decoded credential by name, and do not want to deal with merge conflicts :)